prac_05/word_occurrences.py

```diff
@@ -6,6 +6,19 @@
 user_text = input(str("Please enter a string of text: "))
 
 words = user_text.split()
+#Counting the words from the users input
```

Comment:
When commenting there should be a space after the # before your comment.

Reply:
Totally didn't see that. Thank you.